extension/src/plots/webview/contract.ts

```diff
@@ -3,6 +3,17 @@ import { Color } from '../../experiments/model/status/colors'
 
 export const DEFAULT_NB_ITEMS_PER_ROW = 2
 
+export enum PlotHeight {
+  SMALLER = 0,
+  SMALL = 1,
+  REGULAR = 2,
+  SQUARE = 3,
+  VERTICAL_NORMAL = 4,
+  VERTICAL_LARGER = 5
```

Comment:
[Q] Could these be the aspect ratios if that is what they refer to?

Reply:
Would we gain/lose anything by having this as `PlotAspectRatio`?

Reply:
First iteration used aspect ratios instead, but it made it more complicated to hold fractions instead of numbers. It makes it way easier to change the aspect ratio at the end of the chain (stylesheet) than doing so inside the enum here.